libraries/rush-lib/src/api/VersionPolicy.ts

```diff
@@ -25,8 +25,6 @@ import { cloneDeep } from '../utilities/objectUtilities';
  * This is a copy of the semver ReleaseType enum, but with the `none` value added and
  * the `premajor` and `prepatch` omitted.
  * See {@link LockStepVersionPolicy._getReleaseType}.
- *
- * TODO: Consider supporting `premajor` and `prepatch` in the future.
  */
 export enum BumpType {
```

Comment:
Do the numeric values of these need to be updated to reflect the order of increasing severity? These may be used to ensure that the most severe bump type wins if multiple are specified in a package's changefiles.

Reply:
However, we need to confirm that doing that as a breaking API change won't cause issues.

Maybe we make `premajor` be `4.5` and `prepatch` be `1.5`? When `preminor` was added, was that also a breaking reordering of values? It that's the case, then reshuffling these and using integers is probably fine.

Reply:
@iclanton IMHO it makes sense for them to be ordered in more natural way, but as you mentioned it could be a breaking API change.

I'll wait for other maintainers to comment on this before making changes here.

Reply:
@octogonz - thoughts?